scripts/icecap-git-reset.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
Could we have a comment explaining what this script is for?

Reply:
(It also needs the standard copyright and licensing header at the top of the file.)